src/org/javarosa/xpath/expr/XPathFuncExpr.java

```diff
@@ -1045,14 +1052,22 @@ private static Double round(double number, int numDecimals) {
             return NaN;
         }
 
+        // Rounding with positive decimals
         if (numDecimals >= 0) {
-            // Some locales use ',' instead of '.' as a decimal separator. Since Double.valueOf
-            // doesn't handle comma as the decimal separator, we must ensure a dot will be used by
-            // forcing the US format locale.
-            final NumberFormat nf = NumberFormat.getNumberInstance(Locale.US);
-            nf.setMaximumFractionDigits(numDecimals);
-            nf.setGroupingUsed(false);
-            return Double.valueOf(nf.format(number));
+            try {
```

Comment:
Per the XPath 3.0 specification, rounds up, rounds half up, and rounds negative numbers to zero (mathematically, also up, I suppose).